root/artist/SpecialPurpose.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {l} from '../static/scripts/common/i18n';
+
+import ArtistLayout from './ArtistLayout';
+
+const SpecialPurpose = ({artist}: {artist: ArtistT}) => (
+  <ArtistLayout
+    entity={artist}
+    fullWidth
+    page="special_purpose"
```

Comment:
As with https://github.com/metabrainz/musicbrainz-server/pull/892 - we might decide we don't want to send any page here.